WalletWasabi/Blockchain/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -148,7 +148,7 @@ private ProcessedResult ProcessNoLock(SmartTransaction tx)
 				// if the received transaction is spending at least one input already
 				// spent by a previous unconfirmed transaction signaling RBF then it is not a double
 				// spending transaction but a replacement transaction.
```

Comment:
Comment needs to be changed too, as BIP125 RBF flag is not checked anymore.

Reply:
Done